Orange/widgets/cpewidget.py

```diff
@@ -2,6 +2,9 @@
 
 from Orange.widgets import widget
 from neuropype.engine.common import warn_once
+from neuropype.engine.ports import ListPort
+
+NoneUIValue = ['(use default)', 'default', 'defaults', '(use defaults)']
```

Comment:
In understand your reasoning although I am a bit conflicted by your first and last sentences. Please clarify the options that I should include.